src/Benchmarks.ServerJob/ServerJob.cs

```diff
@@ -35,7 +35,8 @@ public class ServerJob : IIdentifiable
         public int Port { get; set; } = 5000;
         public string Path { get; set; } = "/";
         public string AspNetCoreVersion { get; set; } = "2.1.0-*";
-
+        public Database Database { get; set; } = Database.PostgreSql;
```

Comment:
Would it be better to default this to `None`?  It's the default in `Benchmarks/Configuration/AppSettings.cs`.

Reply:
I didn't want to break any current job definition as the new parameter is not specified for now. And I assumed you'd preferred defaults.

Reply:
If we're going to run all 3 servers in our Cloud environment, there's no reason one server should be the default over the others.  So I would default to `None` and add `PostgreSql` to jobs we want to run on `PostgreSql`, like say the jobs in `build/repo.targets`.